source/displayModel.py

```diff
@@ -280,8 +280,14 @@ def _get__storyFieldsAndRects(self):
 		if not bindingHandle:
 			log.debugWarning("AppModule does not have a binding handle")
 			return [],[],[]
-		left,top=windowUtils.physicalToLogicalPoint(self.obj.windowHandle,left,top)
-		right,bottom=windowUtils.physicalToLogicalPoint(self.obj.windowHandle,right,bottom)
+		try:
+			left,top=windowUtils.physicalToLogicalPoint(self.obj.windowHandle,left,top)
+		except RuntimeError:
+			log.debugWarning("physicalToLogicalPoint failed for top left", exc_info=True)
+		try:
+			right,bottom=windowUtils.physicalToLogicalPoint(self.obj.windowHandle,right,bottom)
+		except RuntimeError:
+			log.debugWarning("physicalToLogicalPoint failed for bottom right", exc_info=True)

```

Comment:
Should not this return from the function? if there is a runtimeError, then  some of left,,top,right or button are going to not exist or be wrong...

Reply:
I'm not sure about this one. Returning from the function results in no text being returned at all. I guess your  opinion is that it might be better to return no text at all rather than the wrong text. That makes sense, but then we really need to log something on the error level.